docker/testrunner/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+cd /black-screen
+
+if [[ ! -e /.firstrun ]] && ( [[ ! -d node_modules ]] || [[ -n $FORCE ]] )
+	then
+	npm install -g selenium-standalone
+	selenium-standalone install
+	chown -R testrunner:users .
+	sudo -u testrunner echo '{ "interactive": false }' > /home/testrunner/.bowerrc
+	sudo -u testrunner npm install
```

Comment:
Is it possible to install the modules somewhere else? I use OS X, and it would make me recompile the local modules every time I use the container.